src/tests/test_time_travel.py

```diff
@@ -4,118 +4,126 @@
 from datetime import datetime
 import select
 import mock
+import pytest
+import sys
 from datetime import datetime as datetime_cls
 
 
 def test_time_patch_set_time():
     with TimeTravel(modules_to_patch=__name__) as t:
         
-        assert time.time() == 0
-        t.clock.time = 3600
-        assert time.time() == 3600
+        assert time.time() == 100000
+        t.clock.time += 360
+        assert time.time() == 100360
 
 
 def test_sleep_patch_sleep():
     with TimeTravel(modules_to_patch=(__name__,)) as t:
         
-        assert time.time() == 0
+        assert time.time() == 100000
         time.sleep(3600)
-        assert time.time() == 3600
+        assert time.time() == 103600
         
-        t.clock.time = 7200
-        assert time.time() == 7200
+        t.clock.time = 107200
+        assert time.time() == 107200
         
         
 def test_datetime_patch_set_time():
     with TimeTravel(modules_to_patch=[__name__]) as t:
         
-        assert datetime_cls.today() == datetime_cls.fromtimestamp(0)
-        t.clock.time = 3600
-        assert datetime_cls.today() == datetime_cls.fromtimestamp(3600)
+        assert datetime_cls.today() == datetime_cls.fromtimestamp(100000)
+        t.clock.time = 86400
```

Comment:
Make the tests not move backward in time.

Reply:
OK